src/Squirrel/UpdateManager.ApplyReleases.cs

```diff
@@ -331,15 +331,17 @@ Task<string> installPackageToAppDir(UpdateInfo updateInfo, ReleaseEntry release)
 
                     toMove.ForEach(ld => {
                         ld.GetDirectories()
-                            .ForEachAsync(subdir => subdir.MoveTo(subdir.FullName.Replace(ld.FullName, target.FullName)))
+                            .ForEachAsync(subdir =>
+                                Utility.Retry(() =>
```

Comment:
Again, anything we've seen to suggest that other exception types should be retried?